sdk/monitor/azure-monitor-query/migration_guide.md

```diff
@@ -0,0 +1,110 @@
+for all languages to drive a consistent experience with established API patterns for all services.
+A set of [Python-specific guidelines](https://azure.github.io/azure-sdk/python/guidelines/index.html) was also introduced to ensure
+that Python clients have a natural and idiomatic feel with respect to the Python ecosystem.
+Further details are available in the guidelines for those interested.
+
+### Cross Service SDK improvements
+
+The modern Azure Monitor Query client library also provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as
+- using the new [`azure-identity`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/README.md) library
+to share a single authentication approach between clients
+- a unified logging and diagnostics pipeline offering a common view of the activities across each of the client libraries
+
+### New Features
+
+We have a variety of new features in the version 1.0 of the Monitor Query library.
+
+- Ability to query a batch of queries with the `LogsQueryClient.query_batch()` API.
+- Ability to configure the retry policy used by the operations on the client.
+- Authentication with AAD credentials using [`azure-identity`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/README.md).
+- Refer to the [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-query/CHANGELOG.md) for more new features, changes and bug fixes.
+
+## Important changes
+
+### The Client
+
+In the interest of providing a more intuitive experience, we've renamed the top level client to query logs to `LogsQueryClient` from `LogAnalyticsDataClient` which can be authenticated using Azure Active Directory(AAD). This client is the single entry point to both query a single query or a batch query.
+
+#### Consistency
+We now have methods with similar names, signature and location to create senders and receivers.
+This provides consistency and predictability on the various features of the library.
+
+### Client constructors and authentication
+
+In `azure-loganalytics` v0.1.0:
+
+```python
+from azure.loganalytics import LogAnalyticsDataClient
+from msrestazure.azure_active_directory import ServicePrincipalCredentials
+
+credential = ServicePrincipalCredentials(...)
+client = LogAnalyticsDataClient(credentials=credential)
+```
+
+In `azure-monitor-query` v1.0.x:
+
+```python
+from azure.monitor.query import LogsQueryClient
+from azure.identity import DefaultAzureCredential
+
+credential = DefaultAzureCredential()
+client = LogsQueryClient(credential=credential)
+```
+
+### Sending a Single Query Request
+
+- In the latest version, `QueryBody` is flattenned. Users are expected to pass the query directly to the API.
+- `timespan` is now a required attribute. This helped to avoid querying over the entire data.
```

Comment:
A small code sample may provide clarity here.

```suggestion
- The `timespan` attribute is now required, which helped to avoid querying over the entire data set.
```